src/asset_loader.h

```diff
@@ -1,34 +1,34 @@
 #pragma once
 
-
 #include "scene/scene.h"
-#include "scene/scene_model.h"
-#include "util/colors.h"
-#include "shaders/shader_manager.h"
-
-
-#include <boost/range.hpp>
-#include <boost/filesystem.hpp>
+#include "model/model.h"
 
 #include <assimp/Importer.hpp>
 #include <assimp/scene.h>
 #include <assimp/postprocess.h>
 #include <iostream>
 
+/*
+	Class for loading all assets on startup.
+*/
+
 class AssetLoader {
 private:
-	map<string, SceneModel*> assets;
+	AssetLoader();
```

Comment:
What's the reasoning for making the assetLoader constructor private and having a static pointer to an instance?